apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -329,12 +329,14 @@
             </tr>
           </thead>
           <tbody>
+            <%const fingerprint_colors = ['red2', 'gray3', 'blue3', 'orange3', 'brown1', 'green3']; %>
             <% log.forEach(function(row, index) { %>
             <tr>
               <td class="text-nowrap"><%= row.formatted_date %></td>
               <td><%= row.auth_user_uid %></td>
               <% if (row.client_fingerprint) { %>
-              <td><a tabindex="0" class="btn btn-xs btn-secondary"
+              <td><a tabindex="0" class="badge color-<%= fingerprint_colors[row.client_fingerprint_number % 6] %>"
```

Comment:
As written, the number disappears on hover:

<img width="37" alt="Screenshot 2024-01-10 at 13 17 44" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/40692381-098e-4590-8bc0-42e99ef9217e">

I'd add the `color-hover` class to ensure that the number stays legible during hover. This class is defined in `colors.css`, e.g.:

https://github.com/PrairieLearn/PrairieLearn/blob/0d727700672d961943a38183ec4c9a4c281f4e7f/apps/prairielearn/public/stylesheets/colors.css#L8

I won't pretend that this is a sensible class name, but oh well!

Reply:
Added the `color-hover` class in commit [9b40a7e](https://github.com/PrairieLearn/PrairieLearn/pull/9084/commits/9b40a7ec5c9ea08cf773fdfb1b0fc49c278a3612) to ensure text stays the same color during hover.